hack/make/test-docker-py

```diff
@@ -14,6 +14,11 @@ source hack/make/.integration-test-helpers
 # TODO re-enable test_attach_no_stream after https://github.com/docker/docker-py/issues/2513 is resolved
 # TODO re-enable test_run_container_reading_socket_ws. It's reported in https://github.com/docker/docker-py/issues/1478, and we're getting that error in our tests.
 : "${PY_TEST_OPTIONS:=--junitxml=${DEST}/junit-report.xml --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_attach_no_stream --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_run_container_reading_socket_ws}"
+
+# build --squash is not supported with containerd integration.
```

Comment:
If there will be more exceptions, it may be good to have the comment include the actual test that was excluded (see comments at line 14, 15); it makes it easier to find back "which one" we're describing. (and where suitable, with a tracking ticket)

Reply:
In case we need more skips we can just add each `--deselect` as a separate line:

```bash
PY_TEST_OPTIONS="$PY_TEST_OPTIONS \
--deselect=tests/integration/api_build_test.py::BuildTest::test_build_squash \
--deselect=tests/integration/api_build_test.py::BuildTest::test_XYZ \
--deselect=tests/integration/api_build_test.py::BuildTest::test_ABC"
```

I think the comment would only duplicate information that's already readable and explicit enough.
Let me know if you think otherwise, I'll add the comment.